### PR TITLE
Show pageAction when history state is updated (closes #151).

### LINKED ISSUE #151: Pulse pageAction does not listen to pushState/popState events.
**[Affected versions]:**
-Firefox 52.0 and up
-Pulse v1.1 RC

**[Affected Platforms]:**
-All Windows
-All Mac
-All Linux

**[Prerequisites]:**
-Pulse v1.1 RC was previously installed on a clean profile.

**[Steps to reproduce]:**
1. Open the browser with the profile from prerequisites.
2. Navigate to youtube.com
3. Switch between video.

**[Expected result]:**
- The Pulse icon is displayed for every link.

**[Actual result]:**
- After clicking on several videos the Pulse icon is no longer displayed.

**[Notes]:**
- This scenarios is easily reproducible with www.vimeo.com and www.youtube.com. I've observed this scenario also on www.cnn.com or www.bbc.com when switching between videos from a webpage.
- There is nothing in the console for this. Sometimes the error `TypeError: _simpleStorage.storage.id is undefined[Learn More]` from issue #144  is displayed.
- [Here](https://drive.google.com/drive/folders/0B6QogV2AJFKidEdQTk1FQzRKam8?usp=sharing) are 2 recordings of the issue.

### PR DESCRIPTION
**Pull request by chuckharmston**

### CODE REVIEW

File: src/lib/page-action.js [hunk 1]
```diff
@@ -10,6 +10,12 @@ export default class PageAction {
     browser.webNavigation.onCompleted.addListener(({ frameId, tabId }) => {
       this.create(frameId, tabId);
     });
+    browser.webNavigation.onHistoryStateUpdated.addListener((
+      { frameId, tabId }
+    ) =>
```
Comment (fzzzy) [inline src/lib/page-action.js:15]: Nit: Don't you think it would look better with the { on this line?

Comment (chuckharmston) [reply src/lib/page-action.js:15]: Debatably yes, but I'm using [prettier](https://prettier.github.io/prettier/) for style, and this is what it does 🤷‍♂️

Comment (fzzzy) [reply src/lib/page-action.js:15]: Uh oh! I had hoped that prettier would do Exactly What I Want! Sigh... Why doesn't everyone in the world have the same taste I do?

**Review by lmorchard — APPROVED** [review]
Works & LGTM

**Pull request merged**